src/views/ViewTrackDetails.tsx

```diff
@@ -115,6 +133,130 @@ export default function ViewTrackDetails() {
             }}
           />
         </Setting.Section>
+        <Setting.Section>
+          <Setting.Input
+            label="Year"
+            id="year"
+            name="year"
+            type="number"
+            min="1900"
+            step="1"
+            value={formData.year}
+            onChange={(e) => {
+              setFormData({ ...formData, year: Number(e.currentTarget.value) });
```

Comment:
I think we need to be more careful here as `e` or `NaN` for example are considered a valid number in JS.

Reply:
Also, we should allow for empty values: pseudo code: `if (value === "") formData.year = null`